lib/DocumentInspectorFactoryInterface.php

```diff
@@ -11,27 +11,29 @@
 
 namespace Sulu\Component\DocumentManager;
 
+use Sulu\Component\DocumentManager\DocumentManagerContext;
+
 /**
  * Document inspector factory.
  *
  * Allows the client to implement their own DocumentInspector implementations.
  *
- * As Document inspectors are dependent upon a document manager, and as the
- * number of document managers is unbounded it would be impractical to manually
- * set the document manager on each individual instance.
+ * As Document inspectors are dependent upon a document context, and as the
+ * number of document contexts is unbounded it would be impractical to manually
+ * set the document context on each individual instance.
  *
  * The factory should create the DocumentInspector instance using the given
- * document manager.
+ * document context.
  */
 interface DocumentInspectorFactoryInterface
 {
     /**
      * Create a new DocumentInspector instance using the given document
-     * manager.
+     * context.
      *
-     * @param DocumentManagerInterface
+     * @param DocumentManagerContext
      *
      * @return DocumentInspector
      */
-    public function getInspector(DocumentManagerInterface $manager);
+    public function getInspector(DocumentManagerContext $context);
```

Comment:
Yes, because we instantiated a single instance.

On Mon, Apr 18, 2016 at 12:03:31AM -0700, Daniel Rotter wrote:
>    In [1]lib/DocumentInspectorFactoryInterface.php:
> 
>  >       *
>  >       * @return DocumentInspector
>  >       */
>  > -    public function getInspector(DocumentManagerInterface $manager);
>  > +    public function getInspector(DocumentManagerContext $context);
> 
>    We also did that until now without a factory, didn't we?
> 
>    —
>    You are receiving this because you authored the thread.
>    Reply to this email directly or [2]view it on GitHub
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu/sulu-document-manager/pull/74#discussion_r60012481
>    2. https://github.com/sulu/sulu-document-manager/pull/74/files/2feed92d47b01ca8229d171cdcabbbbe990012c0#r60012481
>    3. https://github.com/sulu/sulu-document-manager/pull/74/files/2feed92d47b01ca8229d171cdcabbbbe990012c0#r60012481